pkg/kexec-tools/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:db4459afe60aa8affb415cdc370f568fe8bc99bd-dirty
+FROM ${EVE_BUILDER_IMAGE} AS build
+
+ENV BUILD_PKGS patch curl make gcc perl util-linux-dev git mtools linux-headers musl-dev xz-dev elfutils-dev libbz2
+ENV PKGS xz-libs elfutils-dev libbz2
+
+RUN eve-alpine-deploy.sh
+
+# Build kexec-tools
+WORKDIR /tmp/kexec/kexec-tools-2.0.23
```

Comment:
Why is this not an option? https://pkgs.alpinelinux.org/package/edge/testing/x86/kexec-tools

Reply:
Option, but for "edge" branch, not "v3.13". (if I'm not mistaken here with all alpine dependencies)

Reply:
That's not a problem -- since we manually collect binary packages into eve-alpine -- we can get things from whatever branch. Personally -- I'd rather do that.

Reply:
If v3.13 gives you a clear clue of the version and runtime environment, "edge" can be anything. I'm not sure this is a good idea to mix tools from different branches, since tools can be build against different runtime libraries. If this is not the case (I have no idea how alpine is built) - I will change.